package.json

```diff
@@ -7985,5 +8323,16 @@
         "jsonc-parser": "^3.0.0",
         "lodash": "^4.17.21",
         "path-browserify": "^1.0.1"
+    },
+    "__metadata": {
+        "id": "1c81ab96-0424-43c4-b356-fe408a1bd1cf",
+        "publisherId": "60415ab6-4581-4e73-a7e0-6fc6b3369f12",
+        "publisherDisplayName": "VSpaceCode",
+        "targetPlatform": "undefined",
+        "isApplicationScoped": false,
+        "updated": false,
+        "isPreReleaseVersion": false,
+        "installedTimestamp": 1672343009383,
+        "preRelease": false
```

Comment:
what's this `__metadata` field?

Reply:
I didn't notice it.
It has probably been added when I packaged a vsix locally.

I'm going to remove these lines in a new commit.

Reply:
It has been removed in the last commit (490c49bc849364aed8e4facaa0bf84aff911bdd5)